.github/workflows/handle.yml

```diff
@@ -43,9 +43,14 @@ env:
   DEFAULT_DURATION: '2h'
 
 jobs:
-  handle:
+  yaml_lint:
     # Only handle the request when the PR is approved.
     if: '${{ github.event.review.state == ''approved'' }}'
+    uses: 'abcxyz/pkg/.github/workflows/yaml-lint.yml@main' # ratchet:exclude
```

Comment:
Had to rethink whether we should put this job here since users like aod-template cannot ping the SHA of this workflow. 'uses:' does not have access to github context like envs and inputs. Or using main is fine. @yolocs

Reply:
Let's just pin it here.